quickfixj-core/src/main/java/quickfix/Initiator.java

```diff
@@ -30,42 +30,42 @@
      *
      * @see quickfix.SessionFactory#SETTING_CONNECTION_TYPE
      */
-    public static final String SETTING_RECONNECT_INTERVAL = "ReconnectInterval";
+    String SETTING_RECONNECT_INTERVAL = "ReconnectInterval";
```

Comment:
Is there any specific reason to make this package-private? I have also seen it for other constants in this PR.